google-cloud-asset-v1beta1/.repo-metadata.json

```diff
@@ -7,5 +7,6 @@
     "name_pretty": "Cloud Asset V1beta1 API",
     "product_documentation": "https://cloud.google.com/asset-inventory/",
     "repo": "googleapis/google-cloud-ruby",
-    "requires_billing": true
+    "requires_billing": true,
+    "library_type": "GAPIC_MANUAL"
```

Comment:
Sorry, we can drop this one. This library is actually dead, and I've been meaning to delete it.